app/cdap/components/common/TypographyText/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Typography } from '@material-ui/core';
+import { grey, red, green } from '@material-ui/core/colors';
+import { ITypographyTextProps } from 'components/common/TypographyText/types';
+import React from 'react';
+import styled from 'styled-components';
+
+const SimpleText = styled(Typography)`
+  color: ${grey[700]};
+  font-size: ${({ size }) => (size ? size : '14px')};
+  font-weight: ${({ weight }) => (weight ? weight : 500)};
+`;
+
```

Comment:
i think u can refactor these to combine common styles

const SimpleLabel = styled(Typography)`
  font-size: ${({ size }) => (size ? size : '14px')};
  font-weight: ${({ weight }) => (weight ? weight : 500)};
`
then, suppose we need a red card
const RedLabel = styled(SimpleLabel)`
  color: ${grey[700]};
`

plz check if this is achieveable